certified-connectors/Almabase/apiProperties.json

```diff
@@ -0,0 +1,267 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "domain_url": {
+        "type": "string",
+        "uiDefinition": {
+          "constraints": {
+            "clearText": false,
+            "required": true
+          },
+          "description": "Specify your Almabase domain url e.g. https://your.domain.com",
+          "displayName": "Domain URL",
+          "tooltip": "Provide your Almabase domain url e.g. https://your.domain.com"
+        }
+      },
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "The API Key of the user",
+          "tooltip": "Provide your API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": true
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#0942ad",
+    "capabilities": [],
+    "publisher": "Almabase, Inc.",
```

Comment:
@Almabase-Vaibhav Can you please add `stackOwner` as well.

Reply:
Added it. 

When we add stackOwner before running paconn validate, it throws this error. I read this in the limitation of the cli tools.

`{
  "error": {
    "code": "InvalidRequestContent",
    "message": "The request content was invalid and could not be deserialized: 'Could not find member 'stackOwner' on object of type 'ApiPropertiesDefinition'. Path 'properties.stackOwner', line 1, position 655.'."
  }
}
`

Reply:
@diegobernardes Oh, yeah - there is an error in the paconn tool, we are looking into it, thanks for adding manually.